products/compute/terraform.yaml

```diff
@@ -728,6 +728,38 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       decoder: templates/terraform/decoders/snapshot.go.erb
       extra_schema_entry: templates/terraform/extra_schema_entry/snapshot.erb
+  ManagedSslCertificate: !ruby/object:Overrides::Terraform::ResourceOverride
+    docs: !ruby/object:Provider::Terraform::Docs
+      warning: |
```

Comment:
`warning` does `~>` which is a nice yellow warning box, but we could add a new field / modifier to make it `!>` which is a much scarier red box if you'd like. Given the gravity of the message, it could be appropriate.

See https://www.terraform.io/docs/providers/aws/guides/version-2-upgrade.html for an example.

Reply:
Nice, thanks!  I like that idea.  Do you think we have more messages that deserve to be scary boxes instead of warnings?

Reply:
None offhand, most dangerous resources haven't been converted to MM yet.

Reply:
Shrug, then - doesn't seem very important right now, maybe later.  (I see you marked the conversation as resolved, so I'm guessing you agree).